src/verify.v3.integration.test.ts

```diff
@@ -542,8 +542,8 @@ describe("verify v3(integration)", () => {
       expect(valid).toBe(true);
     });
     it("should return valid fragments for documents correctly issued but with data obfuscated", async () => {
-      const obfuscated = obfuscate(v3DidSigned, ["issuer"]);
-      expect(obfuscated.issuer).toBe(undefined);
+      const obfuscated = obfuscate(v3DidSigned, ["reference"]);
```

Comment:
We can't obfuscate any data anymore for testing purposes. If the issuer is missing, the document will be considered invalid.

Reply:
ok noted